metpy/plots/skewt.py

```diff
@@ -465,7 +473,7 @@ def plot_dry_adiabats(self, t0=None, p=None, **kwargs):
         kwargs.setdefault('colors', 'r')
         kwargs.setdefault('linestyles', 'dashed')
         kwargs.setdefault('alpha', 0.5)
-        return self.ax.add_collection(LineCollection(linedata, **kwargs))
+        self.dry_adiabats = self.ax.add_collection(LineCollection(linedata, **kwargs))
```

Comment:
Can you go ahead and end the method with `return self.dry_adiabats` so that we don't change the method interface? Here and the other two methods.

Reply:
Ah, gotcha. Done